ExperimentTwitter/LogInViewController.m

```diff
@@ -0,0 +1,27 @@
+//
+//  LogInViewController.m
+//  ExperimentTwitter
+//
+//  Created by ayur.j on 25/08/17.
+//  Copyright © 2017 ayur.j. All rights reserved.
+//
+
+#import "LogInViewController.h"
+#import "TwitterClient.h"
+#import "TabBarViewController.h"
+
+@implementation LogInViewController
+
+- (IBAction)onLogin:(id)sender {
+   [[TwitterClient sharedInstance] loginWithCompletion:^(User *user, NSError *error) {
+       if(user!=nil) {
+           NSLog(@"Welcome %@", user.name);
```

Comment:
Logs are not visible to the user but are mostly for developers for debugging. So instead of welcome messages, they should be more like "X user logged in".